Jamfile

```diff
@@ -491,6 +491,7 @@ variant test_release : release
 	: <asserts>production <debug-symbols>on
 	  <invariant-checks>full <boost-link>shared <optimization>off
 	  <export-extra>on <debug-iterators>on <threading>multi
+	  <inlining>off
```

Comment:
Yes, brew is independent of the OS X image, and `boost-build` brew formula is deprecated btw (from 2014).